src/app/components/logged/transfers/transfers.component.html

```diff
@@ -1,37 +1,67 @@
-<div class="relative overflow-x-auto shadow-md rounded-lg pt-5">
-  <table class="w-full text-sm text-left rtl:text-right text-bzd-white">
-    <thead class="text-xs text-bzd-white uppercase bg-bzd-orange">
-      <tr>
-        <th scope="col" class="px-6 py-3">ID</th>
-        <th scope="col" class="px-6 py-3">Data e Hora</th>
-        <th scope="col" class="px-6 py-3">Valor</th>
-        <th scope="col" class="px-6 py-3">Tipo</th>
-        <th scope="col" class="px-6 py-3">
-          <span class="sr-only">Edit</span>
-        </th>
-      </tr>
-    </thead>
-    <tbody>
-      @for (transaction of transactionPageable.content; track $index) {
-      <tr class="bg-bzd-orange-light border-b hover:bg-gray-500">
-        <th
-          scope="row"
-          class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap"
-        >
-          {{ transaction.id }}
-        </th>
-        <td class="px-6 py-4">
-          {{ transaction.timestamp | date : "dd/MM/yy - HH:mm" }}
-        </td>
-        <td class="px-6 py-4">R$ {{ transaction.value | number : "1.2-2" }}</td>
-        <td class="px-6 py-4">
-          {{ transaction.type }}
-        </td>
-        <td class="px-6 py-4 text-right">
-          <a href="#" class="bzd-link">Detalhes</a>
-        </td>
-      </tr>
-      }
-    </tbody>
-  </table>
+<div class="w-full">
+  <h1 class="text-3xl font-bold bg-orange-500 text-white p-4 w-full">
+    Meu Histórico
+  </h1>
+
+  <div class="max-w-4xl mx-auto p-4">
+    <div *ngFor="let transaction of transactionPageable?.content" class="bg-white rounded-lg mb-4 border-t-4 border-orange-500 shadow-sm">
```

Comment:
Por questão de padronização, não usa as diretrizes `*ngFor` usa a notação `@for(){}`
Lembra de usar as cores padrão `bzd-`